src/components/DockerFileGenerator/index.jsx

```diff
@@ -96,6 +96,13 @@ function DockerFileGenerator() {
 
 	// Only generate the output if there are no validation errors
         if (Object.keys(validationErrors).length === 0) {
+            const volumesSection = (
+                (!formData.nodeData && !formData.farmerData) ? 
+                `volumes:
+          node-data:
+          farmer-data:` : ""
```

Comment:
yeah makes sense. Although I'd be surprised if someone would be comfortable mounting the node container in their custom folder but looking for the host container somewhere in /var/lib/docker / /Library/Containers/com.docker.docker/Data/vms/0 / %USERPROFILE%\AppData\Local\Docker\wsl\data\ext4.vhdx in linux/mac/windows respectively.... but it can be done easy too to make it universal :)